bluemira/equilibria/run.py

```diff
@@ -363,69 +534,48 @@ class FixedPulsedCoilsetDesign(PulsedCoilsetDesign):
 
     Parameters
     ----------
-    params: ParameterFrame
+    params
```

Comment:
Not sure how I feel about this docstring remaining now that the constructor has been moved to the base class. It's just asking to go out-of-date if/when we change the `PulsedCoilsetDesign`'s constructor. Will sphinx auto-generate the docstring from the parent class's when it sees that this does not override the constructor?

Reply:
yep sphinx is clever enough